redash/handlers/queries.py

```diff
@@ -239,6 +239,17 @@ def delete(self, query_id):
         query.archive(self.current_user)
         models.db.session.commit()
 
+    def patch(self, query_id):
```

Comment:
Feels non idiomatic.. :\ Maybe a regular update (POST) request with `is_archived` set to `false`?